src/js/components/modals/CliInstallModal.js

```diff
@@ -18,9 +18,12 @@ class CliInstallModal extends React.Component {
   constructor() {
     super(...arguments);
 
-    this.state = {
-      selectedOS: browserInfo().os
-    };
+    let selectedOS = browserInfo().os;
```

Comment:
`browserInfo().os` returns a lowercase string right? Because we check `Object.values(osTypes).includes(selectedOS)` here which would only match if it were lowercase. So something is wrong here - either L21 should be mapped to uppercase or we need to lowercase `selectedOS` before we do `Object.values(osTypes).includes(selectedOS)`.